lib/recipes/additive-manufacturing/bake

```diff
@@ -13,10 +13,11 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :additive_manufacturing) do |d
   # Some stuff just goes away
   book.search('cnx-pi').trash
 
+  BakeUnnumberedFigure.v1(book: book)
   BakePreface.v1(book: book)
 
   book.pages('$.preface').each do |page|
-    page.figures.each do |figure|
+    page.figures(only: :figure_to_number?).each do |figure|
```

Comment:
Within this PR I have also updated recipes bake files which were missing `BakeUnnumberedFigure` and didn't use `figure_to_number` for baking numbered figures.